Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -227,9 +232,48 @@ noncomputable instance levyProkhorovDist_pseudoMetricSpace_probabilityMeasure :
 lemma LevyProkhorov.dist_def (μ ν : LevyProkhorov (ProbabilityMeasure Ω)) :
     dist μ ν = levyProkhorovDist μ.toMeasure ν.toMeasure := rfl
 
+/-- A simple sufficient condition for bounding `levyProkhorovEDist` between probability measures
+from above. The condition involves only one of two natural bounds, the other bound is for free. -/
+lemma levyProkhorovEDist_le_of_forall_add_pos_le'
+    (μ ν : Measure Ω) [IsProbabilityMeasure μ] [IsProbabilityMeasure ν] (δ : ℝ≥0∞)
+    (h : ∀ ε B, 0 < ε → ε < ∞ → MeasurableSet B → μ B ≤ ν (thickening (δ + ε).toReal B) + δ + ε) :
```

Comment:
Is there a reason to have this condition in terms of `δ + ε` for positive `ε`, instead of in terms of `ε` for any `δ < ε`, like it is in other lemmas?

Reply:
I changed to the `δ < ε` spellings. It is probably cleaner, indeed.